README.MD

```diff
@@ -93,6 +97,60 @@ Navigate to the folder `infrastructure/local` directory within the clone of the
 
 Execute `docker compose down` to stop the infrastructure containers
 
+# Local Deployment with java -jar
+
+This deployment directly runs mod-inventory-storage-fat.jar.
+
+For a deployment that runs a Docker container that wraps mod-inventory-storage-fat.jar
+see "Local Deployment using Docker" above.
+
+## Start PostgreSQL, Kafka and Zookeeper
+
+This starts them as Docker containers.
+
+    cd run-locally
+    docker-compose up -d
+    cd ..
```

Comment:
Why can this not use the existing, shared, docker compose file for spinning up the infrastructure needed for the module?